### ISSUE TITLE
Azure credits for Open Source Projects

### ISSUE THREAD

**Issue by bnb** [nodejs/build#2796]
I've been asked a number of times if Microsoft would be interested in/willing to give more direct Azure money to the project. There's not really been a framework for this to date, but that recently changed with the [announcement](https://cloudblogs.microsoft.com/opensource/2021/09/28/announcing-azure-credits-for-open-source-projects/) of the [Azure Credits for Open Source Projects](https://opensource.microsoft.com/azure-credits#credits-overview) initiative.

the tl;dr is that if we'd like to apply, Azure is giving a non-trivial amount of free credits to open-source projects. I initially shared this with @mhdawson and he recommended I create an issue here. I'm relatively confident Node.js - and more specifically the Build WG - would be successful in our application.

Happy to answer any questions I can, privately or publicly.

**Comment by mhdawson** [comment]
@joaocgreis do we already use Azure for some of our Windows builds and if so would we want to apply to this for additiona/continuing credits ?

**Comment by mhdawson** [comment]
We have VMs in Azure so I think this can be  closed. Please let us know if that was not the right thing to do.
